api/tests/unit/organisations/test_unit_organisation_views.py

```diff
@@ -1340,3 +1342,91 @@ def test_list_my_groups(organisation, api_client):
         "id": user_permission_group_1.id,
         "name": user_permission_group_1.name,
     }
+
+
+def test_payment_failed_chargebee_webhook(
```

Comment:
This comment may not be relevant if we rely on serialization (since we can then rely on e.g. DRF tests) but these tests are only covering the happy paths, I'd love to see more coverage.

Reply:
I can add more tests. Seems silly sometimes to do a whole test for a single small branch, but it's often worth it.

Reply:
Ok I added more non-happy path tests.